src/components/app-navigator/BottomTabNavigator.tsx

```diff
@@ -66,6 +67,7 @@ const createStackScreen = <StackParamList extends ParamListBase>(
       {baseScreen(Stack)}
       <Stack.Screen name='track' component={TrackScreen} />
       <Stack.Screen name='profile' component={ProfileScreen} />
+      <Stack.Screen name='EditProfile' component={EditProfileScreen} />
```

Comment:
Should we be consistent on the casing for the screen names?

Reply:
yes totally agree, was adding this to get your thoughts cause it seems react-navigation prefers pascal case, thoughts? fine with just doing kebab like we have been tho

Reply:
I actually don't know why I deviated from the react-navigation convention. I'm fine with pascal case!